py/server/deephaven/time.py

```diff
@@ -54,18 +87,7 @@ def now(system: bool = False, resolution: str = 'ns') -> Instant:
         DHError
     """
     try:
-        if resolution == "ns":
-            if system:
-                return _JDateTimeUtils.nowSystem()
-            else:
-                return _JDateTimeUtils.now()
-        elif resolution == "ms":
-            if system:
-                return _JDateTimeUtils.nowSystemMillisResolution()
-            else:
-                return _JDateTimeUtils.nowMillisResolution()
-        else:
-            raise ValueError("Unsupported time resolution: " + resolution)
+        return _JDateTimeUtils.now(system, resolution);
```

Comment:
I'm concerned that this rewrite everything as a single method call is going to be a fail as we get into more complex parts of the API.  I know that we have a lot of things that do branching, possibly in ways where this is not nice.

Reply:
It is really difficult to describe the branching logic in the Python code and then use it to translate to the correct Java code in the QLP, and that's why we want to move these branching logic to the Java API.
Replacing Python wrapper functions with Java ones should be for those wrapper functions that are expected to be used in query formulas.